sketch3.js

```diff
@@ -93,25 +113,28 @@ function drawMap(data) {
                     }
 
 
-                    var nodeX = x + n[3] / 16;
-                    var nodeY = y + n[4] / 16;
+                    var rotated = qrotate(n[3] / 16, n[4] / 16, 0, rx, ry, rz, rw);
+                    var nodeX = x + rotated[0];
+                    var nodeY = y + rotated[1];
                     var _link = n[1];
                     if (_link !== -1 && el.node[t].length > _link) {
 
                         var link = el.node[t][_link];
                         stroke(255);
-                        if (link)
-                            line(px(nodeX), py(nodeY), px(x + link[3] / 16), py(y + link[4] / 16));
+                        if (link) {
+                            var rotatedLink = qrotate(link[3] / 16, link[4] / 16, 0, rx, ry, rz, rw);
+                            line(px(nodeX), py(nodeY), px(x + rotatedLink[0]), py(y + rotatedLink[1]));
+                        }
                     }
-                    ppoint(x + n[3] / 16, y + n[4] / 16);
+                    ppoint(x + rotated[0], y + rotated[1]);
                 });
                 // roadblocks
-                if (ext == 2 && int == 1) {
+                /*if (ext == 2 && int == 1) {
```

Comment:
Actually this code works as expected, it marks a point where a police roadblock can be created in GTA III. They can only appear at the node group with exactly two external links and a single internal link.